kalite/distributed/static/js/distributed/software-keyboard.js

```diff
@@ -106,6 +105,14 @@ window.SoftwareKeyboardView = Backbone.View.extend({
             this.toggle_keypad();
         }
 
+    },
+
+    hide: function() {
```

Comment:
Methods to show and hide the entire software-keyboard view (including the show and hide button). This is so we can turn it off when it is not relevant.